src/PayID/xrp-pay-id-client.ts

```diff
@@ -23,6 +26,19 @@ export default class XRPPayIDClient extends PayIDClient {
    */
   async xrpAddressForPayID(payID: string): Promise<string> {
     const result = await super.addressForPayID(payID)
-    return result.address
+
+    const { address } = result
+    if (Utils.isValidXAddress(result.address)) {
```

Comment:
you can just use `address` here since you destructured it already

Reply:
ty